app/views/sufia/admin_sets/_admin_set.html.erb

```diff
@@ -21,5 +21,5 @@
   </td>
   <td><%= admin_set.create_date %></td>
   <td>N/A</td>
-  <td>N/A</td>
+  <td><%= controller.presenter_class.new(admin_set, current_ability).total_items %></td>
```

Comment:
Ya, forgot that was in there. This line should have been included in a previous PR made. I'll pull it put and work it through a new issue.